DEVELOP.md

```diff
@@ -60,7 +60,7 @@ certbot_dns_loopia-whl/
 │   ├── WHEEL
 │   ├── entry_points.txt
 │   └── top_level.txt
-└── certbot_dns_loopia.py
+└── dns_loopia.py
```

Comment:
I am guessing this file structure visualization needs to be updated. I have not tried building or running twine yet.